lib/_http_incoming.js

```diff
@@ -123,7 +123,7 @@ ObjectDefineProperty(IncomingMessage.prototype, 'headers', {
     return this[kHeaders];
   },
   set: function(val) {
-    this[kHeaders] = val;
+    noop();
   }
 });
```

Comment:
Would this work instead?

```suggestion
  set: function(val) {}
```